src/unity/python/turicreate/visualization/plot.py

```diff
@@ -107,13 +107,18 @@ def save(self, filepath, include_data=True):
 
         Parameters
         ----------
+        filepath: string
+            The destination filepath where the plot object must be saved as.
+            The extension of this filepath determines what format the plot will
+            be saved as. Currently supported formats are JSON, PNG, and SVG.
         include_data : bool, optional
             If True, save's the Plot in a vega spec with the data spec
-            included.
+            included. This parameter is not important when saving as a PNG/SVG
 
         Notes
         -----
-        - The save method saves the Plot object in a vega json format
+        - The save method saves the Plot object in a vega json format, or a 
```

Comment:
I think this could be worded more clearly - but is redundant with the parameters/description above anyway. I'd remove this entire `Notes` section.

Reply:
Done!